.github/workflows/NativeAutomatedTestsAndroid.yml

```
@@ -0,0 +1,96 @@
+name: Run native automated end-to-end tests for Android
+
+on:
+    workflow_dispatch
+    # push:
+    #     branches: [master]
+    #     paths: "packages/pluggableWidgets/*-native/**/*"
+
+    # pull_request:
+    #     branches: [master]
+    #     paths: "packages/pluggableWidgets/*-native/**/*"
+
+jobs:
+    native_automated_tests_android:
+        name: "Native automated tests for Android"
+        runs-on: ubuntu-latest
+        permissions:
+            packages: read
+            contents: read
+        steps:
+            #     - name: "Checking changed files"
+            #       id: files
+            #       uses: softprops/diffset@v1
+            #       if:
+            #           github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name ==
+            #           github.repository
+            #       with:
+            #           global_files: |
+            #               !{packages/**/*,configs/e2e/*.json,detox/**/*,scripts/**/*}
+            #           not_atlas_native_files: |
+            #               !packages/theming/atlas/src/**/native/**
+            #       env:
+            #           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+            - name: "Checking-out code"
+              uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+              with:
+                  submodules: false
+            - name: "Fetching master to compare"
+              if:
+                  github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name !=
+                  github.repository
+              run:
+                  git fetch --no-tags --prune --depth=1 origin +refs/heads/${{ github.base_ref
+                  }}:refs/remotes/origin/${{ github.base_ref }}
+            # - name: "Defining environment variables"
+            #   id: variables
+            #   run:
+            #       echo ::set-output name=arg::$(if [ "${{ steps.files.outputs.global_files }}" = "" ] && ${{
+            #       github.event_name == 'pull_request' }}; then echo '--since origin/${{ github.base_ref }}'; else echo
+            #       ''; fi)
+            - name: "Defining cache"
+              uses: actions/cache@937d24475381cd9c75ae6db12cb4e79714b926ed # v2
+              env:
+                  cache-name: cache-node-modules
+              with:
+                  path: ~/.npm
+                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
```

Probably you need to change this hashFiles to search the package-lock.json in the root, because that started to show an error when you use this relative path. I did that to fix that error in the cache action yesterday.

```suggestion
                  key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('package-lock.json') }}
```